Admin/AdminInterface.php

```diff
@@ -308,6 +308,21 @@ public function getIdParameter();
     public function hasRoute($name);
 
     /**
+     * Check the current request is given route or not.
+     *
+     * ```
+     * $this->isCurrentRoute('create'); // is create page?
+     * $this->isCurrentRoute('edit', 'some.admin.code'); // is some.admin.code admin's edit page?
+     * ```
+     *
+     * @param string $name
+     * @param string $adminCode
+     *
+     * @return bool
+     */
+    public function isCurrentRoute($name, $adminCode = null);
```

Comment:
@greg0ire if we leave this as it is, we only can merge this in the next major, otherwise, @Rivsen has to comment this